test/unit/spec/MfaVerify_spec.js

```diff
@@ -114,7 +118,7 @@ function (Okta,
     'GENERIC_SAML': 14,
     'GENERIC_OIDC': 15
   };
-
+  /* eslint max-statements: [2, 50]*/
```

Comment:
isn't this override working https://github.com/okta/okta-signin-widget/blob/master/test/unit/.eslintrc.js#L10?